.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: go
 
 go:
-  - '1.11.x'
+  - '1.12.x'
```

Comment:
Perhaps 1.13?

Reply:
Thanks, @slaskawi 
I will change this for travis build